docs/mkdocs.yml

```diff
@@ -41,9 +41,10 @@ nav:
   - how-to/create-aws-hosted-cluster-multiple-zones.md
   - how-to/deploy-aws-private-clusters.md
   - how-to/oauth.md
-  - how-to/agent-platform.md
   - how-to/create-azure-cluster.md
   - how-to/kubevirt-platform.md
+  - how-to/baremetal-none.md
```

Comment:
can be consistent here and use the therm baremetal either as a prefix or a suffix in both places?
e.g baremetal-none, baremetal-agent.

Reply:
Fixed. Thanks.